### DISCUSSION TITLE
azure-boards-tips-operations-team

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#30] [Technical Blog]
# azure-boards-tips-operations-team

https://wsbctechnicalblog.github.io/azure-boards-tips-operations-team.html

**Comment by parkay909** [comment]
Which default process did you use for this board?

**Reply by wpschaub** [reply]
I used a customized process, which is based on the out-of-the-box Agile process.
